ramls/receiving_history.raml

```diff
@@ -0,0 +1,32 @@
+#%RAML 1.0
+title: "mod-orders-storage"
+baseUri: http://github.com/folio-org/mod-orders-storage
+version: v1
+
+documentation:
+  - title: Receiving History
+    content: <b>Get list of receiving history API</b>
+
+types:
+    receiving_history: !include acq-models/mod-orders-storage/schemas/receiving_history.json
+    receiving_history_collection: !include acq-models/mod-orders-storage/schemas/receiving_history_collection.json
+
+traits:
+    pageable:  !include raml-util/traits/pageable.raml
+    searchable: !include raml-util/traits/searchable.raml
+    language: !include raml-util/traits/language.raml
+
+resourceTypes:
+    collection-get: !include raml-util/rtypes/collection-get.raml
+
+/orders-storage/receiving_history:
```

Comment:
As discussed in the Slack, we are going to start to follow [FOLIO Naming conventions](https://dev.folio.org/guidelines/naming-conventions/#api-endpoints). So probably it makes sense to start applying them at least for the new changes:
> Endpoints use dash-separated strings, with URI parameters as camelCase.